### ISSUE TITLE
Invalid Schema Name XXXXXXXXXX

### ISSUE THREAD

**Issue by mohammadsiddiqui** [SAP/node-hdb#137]
I need to access the tables defined in an HDI container, I am not able to make a query on those tales, i keep getting `invalid schema name`.

**Comment by alexpenev-s** [comment]
Hi, Could you please provide a repro scenario?

**Comment by mohammadsiddiqui** [comment]
I have a HANAExpress 2.0 and have created a CDS views from WebIDE which which results into table in  HDI Container. I want to access the data from these tables.

**Comment by alexpenev-s** [comment]
Do if you use the hana-client driver do you get the same error?
(Please see top of the README for install instructions)

**Comment by mohammadsiddiqui** [comment]
Yes @alexpenev-s I have used both node-hdb and hana-client and error is same in both the cases.

**Comment by alexpenev-s** [comment]
It's likely that the problem is with HANA - are you able to see the HANA logs for more information?
